openquakeplatform/bin/oq_install.sh

```diff
@@ -0,0 +1,226 @@
+    geonodedir=`python -c "import geonode;import os;print os.path.dirname(geonode.__file__)"`
+
+    ln -sf /etc/geonode/local_settings.py $HOME/env/lib/python2.7/site-packages/geonode/local_settings.py
+    ln -sf /usr/lib/python2.7/dist-packages/osgeo $HOME/env/lib/python2.7/site-packages
+
+    # Set up logging symlink
+    mkdir -p $GEONODE_LOG
+    ln -sf /var/log/apache2/error.log $GEONODE_LOG/apache.log
+
+    export DJANGO_SETTINGS_MODULE=geonode.settings
+
+    unsudo 'source env/bin/activate ; django-admin migrate account --settings=geonode.settings'
+    unsudo 'source env/bin/activate ; geonode migrate --verbosity 0'
+    unsudo 'source env/bin/activate ; geonode loaddata $geonodedir/base/fixtures/initial_data.json'
+    unsudo 'source env/bin/activate ; geonode collectstatic --noinput --verbosity 0'
+
+    if [ -z "$DEVEL_DATA" ]; then
+        unsudo 'source env/bin/activate ; geonode createsuperuser'
+    fi
+
+    # ipt folder
+    cd $GEONODE_WWW
+    mkdir data
+    chown www-data.www-data -R $GEONODE_WWW/data
+    chmod 775 -R $GEONODE_WWW/data
+    if [ "$DEVEL_DATA" ] || [ "$DATA_PROD" ]; then
+        chmod g+s $GEONODE_WWW/data
+    fi
+}
+
+function setup_apache_once() {
+    chown www-data -R $GEONODE_WWW
+    a2enmod proxy_http
+
+    sed -i '1d' $APACHE_SITES/geonode.conf
+    sed -i "1i WSGIDaemonProcess geonode user=www-data threads=15 processes=2" $APACHE_SITES/geonode.conf
+    sed -i '1 s@^@WSGIPythonHome /home/openquake/env\n@g' $APACHE_SITES/geonode.conf
+
+    #FIXME: This could be removed if setup_apache_every_time is called after setup_apache_once
+    $APACHE_SERVICE restart
+}
+
+function setup_apache_every_time() {
+    a2dissite 000-default
+
+    #FIXME: This could be removed if setup_apache_every_time is called after setup_apache_once
+    a2enmod proxy_http
+
+    a2ensite geonode
+    $APACHE_SERVICE restart
+}
+
+function setup_geoserver() {
+    cd $HOME/geonode
+    wget http://ftp.openquake.org/oq-platform2/geoserver-2.9.x-oauth2.war
+    mv geoserver-2.9.x-oauth2.war geoserver.war
+    mv geoserver.war $TOMCAT_WEBAPPS
+
+    $TOMCAT_SERVICE restart
+
+    ## Symbolic link to solve spatialite warning of Geoserver
+    sudo ln -sf /usr/lib/x86_64-linux-gnu/libproj.so.9 /usr/lib/x86_64-linux-gnu/libproj.so.0
+}
+
+function postinstall() {
+    setup_postgres_every_time
+    setup_django_every_time
+    setup_apache_every_time
+    $TOMCAT_SERVICE restart
+    $APACHE_SERVICE restart
+}
+
+function once() {
+    echo "Still need to implement the onetime setup."
+    exit 1
+}
+
+if [ $# -eq 1 ]
+then
+    printf "Sourcing %s as the configuration file\n" $1
+    source $1
+else
+    printf "Usage: %s: [-s value] configfile\n" $(basename $0) >&2
+        exit 2
+fi
+
+if [ "$stepflag" ]                                                              
+then
+printf "\tStep: '$stepval specified\n"
+else
+    stepval="all"
+    echo "heh"
```

Comment:
what does it means this ``echo`` ?

Reply:
Was there already .. I delete him?

Reply:
Some changes here: https://github.com/gem/oq-platform2/pull/66/commits/554ec877c59b5378a3a6f3e22342d54ae15a7c8d